salesforce-connector-demo/src_hd/salesforce/connector/demo/Chart/Chart.xhtml

```diff
@@ -0,0 +1,64 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/basic-10.xhtml">
+		<ui:define name="title">Chart</ui:define>
+		<ui:define name="content">
+			<h:form id="form">
+				<div class="ui-fluid">
+					<div class="ui-g">
+						<div class="ui-g-12 ui-md-12 ui-lg-12">
+					        <div class="card">
+					            <p:barChart model="#{data.bean.barModel}" style="width: 100%; height: 600px;"/>
+					        </div>
+					    </div>
+					</div>
+				</div>
+				<br />
+				<br />
+				
+				<div class="command-btns">
+					<p:commandButton id="proceed" actionListener="#{logic.close}"
+						value="Close" update="form" icon="si si-close" />
+				</div>
+			</h:form>
+			
+			<style>
+				.textDecorationNone {
```

Comment:
ivy embedded the PrimeFlex-v2 https://github.com/primefaces/primeflex/wiki/PrimeFlex-v2
could you try to use it instead of introducing new CSS here, @HoDacVu90